base/scripts/code.game.cs

```diff
@@ -1078,3 +1087,42 @@ function remoteSetArmor(%player, %armorType)
   }
   
 }
+
+function Game::BodyBlockCheck()
+{
+    
+    if(!$BodyBlock::Enabled) { return; }
+    
+    //if we in a pause, back out and try again in 1 second
+    if ($freezedata::actice == 1) {
+        schedule("Game::BodyBlockCheck();", 1);
+        return;
+    }
+    
+    //only look for player speed if the match has begun
+    if ($matchStarted) {
+        
+        //cycle through clients
+        for(%cl = Client::getFirst(); %cl != -1; %cl = Client::getNext(%cl)) {
+            if ($PlayerHasSpawned[%cl]) {
+                %clTeam = Client::getTeam(%cl);
+                if (%clTeam == 0 || %clTeam == 1) {
+                    %otherTeam = (%clTeam + 1) % 2; //if 0 returns 1, if 1 returns 0.
+                    //check to see if we are in the process of calculating a BB first
+                    if (!$BodyBlock::Calculate[%cl]) {
+                        //find speed
+                        $BodyBlock::Speed[%cl] = Game::BodyBlockDistance(%cl, %otherTeam);

```

Comment:
it looks like this function doesn't use %otherTeam at all - could you remove it from the code / the function signature?

Reply:
id like to leave it for now, but will clean it up later given body blocking runs smoothly for a week or so of testing

Reply:
for code hygiene i think it's best if we remove it. it's not hard to put to back and we always have git history if we need to dig old code up again. there's a programming principle called "you aren't gonna need it" (YAGNI) which essentially means that it's a poor practice to leave around code "just in case".

- https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it
- https://martinfowler.com/bliki/Yagni.html

Reply:
done